docs/backup_and_restore.rst

```diff
@@ -560,9 +560,10 @@ source accounts, and journalist accounts. To do so, follow the steps below:
    The new instance's onion service addresses will be unchanged, but the
    old instance's data and accounts will now be available.
 
-#. As part of this process, your .onion URLs have changed, and *Journalist* and
-   *Admin Workstations* will be out of date, and will need to be
-   :ref:`updated <update_tails_v3>`.
+#. As part of this process, your .onion URLs have changed, and any old
+   *Journalist* and *Admin Workstations* will no longer work. To create
+   additional  *Journalist Workstations*, see our
+   :doc:`onboarding documentation <onboarding>`.
```

Comment:
It'd be nice to be able to point to a guide for onboarding admins here, but we don't have one yet (see #256). As is, I think we're better off to remove the instructions for repairing old v2 drives at this point; that process will be difficult to support long term.